src/runtime/inertia/components/Link.vue

```diff
@@ -110,72 +110,53 @@ const linkClass = computed(() => {
 })
 
 const page = usePage()
-const url = computed(() => props.to ?? props.href ?? '#')
+const url = computed(() => props.to ?? props.href ?? '')
 
 const isActive = computed(() => props.active || (props.exact ? url.value === props.href : page?.url.startsWith(url.value)))
 </script>
 
 <template>
-  <template v-if="!isExternal">
-    <InertiaLink v-bind="routerLinkProps" :href="url" custom>
-      <template v-if="custom">
-        <slot
-          v-bind="{
-            ...$attrs,
-            as,
-            type,
-            disabled,
-            href: url,
-            active: isActive
-          }"
-        />
-      </template>
-      <ULinkBase
-        v-else
-        v-bind="{
-          ...$attrs,
-          as,
-          type,
-          disabled,
-          href: url,
-          active: isActive
-        }"
-        :class="linkClass"
-      >
-        <slot :active="isActive" />
-      </ULinkBase>
-    </InertiaLink>
-  </template>
-
-  <template v-else>
-    <template v-if="custom">
-      <slot
-        v-bind="{
-          ...$attrs,
-          as,
-          type,
-          disabled,
-          href: to,
-          target: isExternal ? '_blank' : undefined,
-          active: isActive
-        }"
-      />
-    </template>
-    <ULinkBase
-      v-else
+  <InertiaLink v-if="!isExternal && custom && !!url" v-bind="routerLinkProps" :href="url">
```

Comment:
@jvbianchi Wouldn't something like this be more straightforward? https://github.com/nuxt/ui/commit/0d8adf782db7ae5303cc7c0c0817c3167c223497

Reply:
I think you are right. Should I close this PR?

Reply:
No let's update this one :)

Reply:
I updated this PR with you change